src/test/resources/fixtures/hmh/hmh-rce/development/caliperEventNavigationEvent.json

```diff
@@ -0,0 +1,75 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "id": "urn:uuid:c942788b-aae8-4a15-8135-b5ec731a0fc3",
+  "type": "NavigationEvent",
+  "actor": {
+    "id": "urn:uuid:ff7c53fd-9024-4d2d-8503-aa3da2cbc73e",
+    "type": "Person"
+  },
+  "action": "NavigatedTo",
+  "object": {
+    "id": "http://rce.localhost/content/book/OPS/page0001.xhtml",
+    "type": "Frame",
+    "name": "Cover Page",
+    "dateCreated": "2019-06-05T14:17:34.739Z",
```

Comment:
I would drop `dateCreated` as it is the date created of the content object which we don't care about and it is optional.

Reply:
Cool